evm_loader/faucet/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "faucet"
+description = "Ethereum ERC20 Token Faucet Service"
+authors = ["NeonLabs Maintainers <maintainers@neonlabs.org>"]
+repository = "https://github.com/neonlabsorg/neon-evm/evm_loader/faucet"
+version = "0.2.2"
+build = "build.rs"
+edition = "2018"
+
+[dependencies]
+actix-cors = { version = "0.6.0-beta.2", default-features = false }
+actix-web = { version = "4.0.0-beta.9", default-features = false }
+color-eyre = "0.5"
+derive-new = "0.5"
+ed25519-dalek = "1.0"
+evm-loader = { path = "../program", default_features = false, features = ["no-entrypoint"] }
+hex = "0.4"
+lazy_static = "1.4"
+nix = "0.22"
+num_cpus = "1.13"
+secp256k1 = "0.20"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+solana-client = "=1.7.10"
+solana-sdk = "=1.7.10"
+spl-associated-token-account = { version = "1.0", default_features = false, features = ["no-entrypoint"] }
+spl-token = { version = "3.2", default_features = false, features = ["no-entrypoint"] }
+structopt = "0.3"
+thiserror = "1.0"
+tokio = { version = "1.10", default_features = false, features = ["rt"] }
+toml = "0.5"
+tracing = "0.1"
+tracing-subscriber = "0.2"
+web3 = { path = "web3" }
```

Comment:
Why we can't use `web3` package from the rust repository?

Reply:
It was because of gasPrice. I started with the public version of web3, of course, but had to modify it to support non-standard result of the proxy's gasPrice at that time. Probably we should try to return to the public version of web3 crate.